src/main/java/atrium/AtriumClient.java

```diff
@@ -0,0 +1,526 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.URL;
+import javax.net.ssl.HttpsURLConnection;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+
+import com.google.gson.Gson;
+
+import com.google.gson.JsonParser;
+import com.google.gson.JsonElement;
+
+public class AtriumClient {
+  private String environment;
+  private String mxAPIKEY;
+  private String mxCLIENTID;
+
+  // Required Parameters: environment, mxAPIKEY, mxCLIENTID
+  // Optional Parameters: None
+  public AtriumClient(String environment, String mxAPIKEY, String mxCLIENTID) {
+    this.environment = environment;
+    this.mxAPIKEY = mxAPIKEY;
+    this.mxCLIENTID = mxCLIENTID;
+  }
+
+
+  // USER
+
+
+  // Required Parameters: None
```

Comment:
You should switch these to javadoc comments. Most IDE's use these comments for autocomplete. 

```
/**
   * This method is used to create a User.
   * @param identifier Optional. Param used to identify user.
   * @param isDisabled  Optional. Sets the disabled flag for the user. Defaults to false.
   * @param metadata  Optional. Sets the disabled flag for the user. Defaults to false.
   * @return User This returns the newly created User object.
   */
```